frappe/core/doctype/file/file.py

```diff
@@ -933,6 +933,9 @@ def attach_files_to_document(doc, event):
 	the file url to the document if it is not already attached.
 	"""
 
+	if frappe.flags.in_web_form:
```

Comment:
I presume this is a fix for https://github.com/frappe/frappe/pull/11677#pullrequestreview-511510556 But it really does not solve the problem at the root